components/x-live-blog-wrapper/src/LiveBlogWrapper.jsx

```diff
@@ -29,13 +30,21 @@ const liveBlogWrapperActions = withActions({
 	}
 });
 
-const LiveBlogWrapper = liveBlogWrapperActions(({ posts = [], articleUrl, showShareButtons }) => {
-	const postElements = posts.map((post) => <LiveBlogPost key={`live-blog-post-${post.postId}`} {...post} articleUrl={articleUrl} showShareButtons={showShareButtons}/>);
-	return (
-		<div className='x-live-blog-wrapper'>
-			{postElements}
-		</div>
-	);
-});
 
-export { LiveBlogWrapper };
+const LiveBlogWrapper = liveBlogWrapperActions(function LiveBlogWrapper ({ posts = [], articleUrl, showShareButtons }) {
```

Comment:
I'm a bit confused by this line and how `LiveBlogWrapper` calls itself here but let me take a look at `liveBlogWrapperActions`!